Templates/DefaultProject/Template/ExportScripts/export_standalone_monolithic.py

```diff
@@ -0,0 +1,321 @@
+
+    if should_build_game_launcher:
+        process_command(["cmake", "--build", mono_build_path, "--target", f"{project_name}.GameLauncher", "--config", build_config, "--", "/m"])
+    
+    if should_build_unified_launcher:
+        process_command(["cmake", "--build", mono_build_path, "--target", f"{project_name}.UnifiedLauncher", "--config", build_config, "--", "/m"])
+
+
+def kill_existing_processes(project_name: str):
+    o3de_process_names = ['o3de', 'editor', 'assetprocessor', f'{project_name.lower()}.serverlauncher', f'{project_name.lower()}.gamelauncher' ]
+    for process in psutil.process_iter():
+        processes_to_kill = []
+        # strip off .exe and check process name
+        if os.path.splitext(process.name())[0].lower() in o3de_process_names:
+            processes_to_kill.append(process)
+        
+        if processes_to_kill:
+            logging.error("The following processes are still running: "+[process.name() for process in processes_to_kill])
+            user_input = input(f'Continuing may cause build errors.\nStop them before continuing? (y/n). Quit(q)')
+            if user_input.lower() == 'y':
+                for p in processes_to_kill:
+                    p.terminate()
+                    p.wait()
+            elif user_input.lower() == 'q':
+                quit()
+
+
+def process_assets( project_name: str,
+                    project_path: pathlib.Path,
+                    non_mono_build_path: pathlib.Path):
+    process_command(["cmake", "-B", non_mono_build_path, "-S", project_path])
+
+    process_command(["cmake", "--build", non_mono_build_path, "--target", "AssetProcessorBatch", "AssetBundlerBatch", "--config", "profile", "--", "/m"])
+    process_command(["cmake", "--build", non_mono_build_path, "--target", f"{project_name}.Assets", "--config", "profile", "--", "/m"])
+
+def bundle_assets(project_path: pathlib.Path,
+                  selected_platform: str,
+                  seedlist_paths: List[str],
+                  non_mono_build_path: pathlib.Path):
+    asset_bundler_batch_path = non_mono_build_path / 'bin' / 'profile' / ('AssetBundlerBatch' + ('.exe' if platform.system().lower()=='windows' else ''))
+    asset_list_path = project_path / 'AssetBundling' / 'AssetLists'
+
+    game_asset_list_path = asset_list_path /  f'game_{selected_platform}.assetlist'
+    engine_asset_list_path = asset_list_path / f'engine_{selected_platform}.assetlist'
+    bundles_path = project_path / 'AssetBundling' / 'Bundles'
+
+    #generate the asset lists for the project
+    gen_game_asset_list_command = \
+        f"{asset_bundler_batch_path} assetLists --assetListFile {game_asset_list_path} --platform {selected_platform} --allowOverwrites"
+    for seed in seedlist_paths:
+        gen_game_asset_list_command += " --seedListFile " + str(seed)
+    process_command(gen_game_asset_list_command.split())
```

Comment:
Why are we concatenating to a string only to split it into an array anyways? `gen_game_asset_list_command` can just be initialized as a list and passed into `process_command` directly. If `asset_bundler_batch_path` has a space in it (even if its surrounded by double quotes) will be split as well and will cause problems with the command exec.

```suggestion
    #generate the asset lists for the project
    gen_game_asset_list_command = [
        asset_bundler_batch_path,
        'assetLists',
        '--assetListFile',
        game_asset_list_path,
        '--platform',
        selected_platform,
        '--allowOverwrites']
 
    for seed in seedlist_paths:
        gen_game_asset_list_command.append("--seedListFile")
        gen_game_asset_list_command.append(str(seed))

    process_command(gen_game_asset_list_command)
```